history-of-the-quakers-v2/original/08-book-3-chapter-14.adoc

```diff
@@ -25,26 +31,29 @@ pestilence, or any symptom of divine displeasure, were calamities of equal magni
 with the existence of non-conformity, they proceeded to enact a fresh penal law,
 commonly known by the name of The Oxford five-mile act,
 which received the royal assent October the 31st, 1665.
-It was entitled, An aft to restrain non-conformists from inhabiting corporations;
-the preamble to which sets forth,
+It was entitled, An Act to Restrain Non-conformists from Inhabiting Corporations;
+the reamble to which sets forth,
 
-"`that divers Parsons and others in holy orders,
+[.embedded-content-document.legal]
+--
+
+that divers Parsons and others in holy orders,
 not having subscribed the act of uniformity,
 have taken upon them to preach in unlawful assemblies,
 and to instil the poisonous principles of schism and rebellion
 in the hearts of his Majesty`'s subjects to the church and kingdom:
 Be it therefore enacted,
 that all such non-conformist ministers shall take the following oath,
 
-"` I, A. B. do swear that it is not lawful upon any pretence whatsoever;
+I, A. B. do swear that it is not lawful upon any pretence whatsoever;
```

Comment:
sorry, lots of comments today. :)  this is a weird doc, it's quoting an oath in the middle of the legal doc.  In the original it's italicized, which I think makes sense to set it apart from the rest of the embed. So, can you wrap this whole paragraph in `__` to make it italics? Thanks!